test/e2e/metrics.go

```diff
@@ -0,0 +1,58 @@
+package e2e
+
+import (
+	"context"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = ginkgo.Describe("[RTE] metrics", func() {
+	var (
+		initialized bool
+		rtePod      *v1.Pod = nil
+	)
+
+	f := framework.NewDefaultFramework("metrics")
+
+	ginkgo.BeforeEach(func() {
+		if !initialized {
+			ds, err := f.ClientSet.AppsV1().DaemonSets(defaultNamespace).Get(context.TODO(), rteDsName, metav1.GetOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			pods, err := f.ClientSet.CoreV1().Pods(defaultNamespace).List(context.TODO(), metav1.ListOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			for _, pod := range pods.Items {
+				for _, or := range pod.GetOwnerReferences() {
```

Comment:
or just lookup by a well-known (and documented) label, adding it if needed

Reply:
So add a label in the manifest of the ds? what is the name of the label should look like?

Reply:
We can add something like
```
  template:
    metadata:
      labels:
        name: resource-topology
        name: topology-updater
```
in https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/master/manifests/resource-topology-exporter-ds.yaml#L10

and here in the test just look for a pod labeled `topology-updater` (AND `resource-topology`?). But can be done later, not blocking for this PR.

Reply:
Looking into it right now. address all the others

Reply:
We can't have the same key twice, but anyway I see there is only one pod (per node) with the label ```name=resource-topology``` so this is enough I think